iterations_runners/iterations_runner.lua

```diff
@@ -103,7 +101,7 @@ end
 
 dofile(BM_benchmark)
 
-krun_init()
```

Comment:
Let's cache this one too and only use `krun_done` directly.

Reply:
I didn't think there was much reason to cache it?

Reply:
Well, there isn't, but it makes clear that the `krun_done` case is the only special case.